backend/app/email_service.py

```diff
@@ -163,3 +171,51 @@ def admin_alert_email(self, subject, message, alert_principal_investigator=False
 
             self.send_email(subject,
                             recipients=recipients, text_body=text_body, html_body=html_body)
+
+    def complete_registration_prompt_email(self, user, current_studies):
+        with self.app.app_context(), self.app.test_request_context():
+            tracking_code = self.tracking_code()
+
+            subject = "Autism DRIVE: Complete Your Registration"
+            logo_url = self.api_url + '/api/track/' + str(user.id) + '/' + tracking_code + '/UVA_STAR-logo.png'
```

Comment:
I think Flask provides ways to get a known url endpoint that might dry this up a bit.  Nothing to hold up a push, but something to look into.